### ISSUE TITLE
Vulcanize use of jsoup breaks Polymer templates inside tables

### ISSUE THREAD

**Issue by gar1t** [tensorflow/tensorboard#502]
This is a pretty edge case issue, but from what I'm seeing it is impacting TensorBoard.

The root issue is described here:

- https://github.com/jhy/jsoup/issues/941
- https://github.com/jhy/jsoup/issues/600

You can see the bug manifest in the vulcanized output in `index.html` (snippet):

```
 <template is="dom-repeat" items="[[_getDevices(devicesForStats)]]">
          </template><table>
        <tr>
            <td>
              <input type="checkbox" value$="[[item.device]]" checked$="[[item.used]]" on-click="_deviceCheckboxClicked">
...
```

Note the all important `template` tag has been moved outside the table, breaking the component. The original snippet is [here](https://github.com/tensorflow/tensorboard/blob/master/tensorboard/plugins/graph/tf_graph_controls/tf-graph-controls.html#L400-L404).

As this is impacting my work on plugins, I will look into a fix that I can make available.

**Comment by wchargin** [comment]
Yeah, we've hit this in the past, and summarized it as "Polymer's `dom-*` do not work with tables." Interesting to know that it's vulcanization and not Polymer proper. This explains why it's not reproducible in a polybin, because no vulcanization is performed.

We always just work around this. See, e.g., #191 and #319. An actual fix would be greatly appreciated.

cc @chihuahua
